src/wp-includes/block-supports/shadow.php

```diff
@@ -52,7 +52,10 @@ function wp_register_shadow_support( $block_type ) {
 function wp_apply_shadow_support( $block_type, $block_attributes ) {
 	$has_shadow_support = block_has_support( $block_type, 'shadow', false );
 
-	if ( ! $has_shadow_support ) {
+	if (
```

Comment:
Would be good to add a `since ` note to the docblock above; something like

` @since 6.6.0 Return early if __experimentalSkipSerialization is true.`

(it should be added just below `@since 6.3.0`)